scripts/gdb/lvglgdb/lvgl.py

```diff
@@ -12,31 +13,29 @@
 g_lvgl_instance = None
 
 
-class LVList:
+class LVList(Value):
```

Comment:
Should it be Union[gdb.Value, Value]?

Reply:
I don't wanna accept `gdb.Value`, though the code now supports `gdb.Value`.

All value passed here should be packed with `lvgl.Value`